front/app/components/UI/FormComponents/index.tsx

```diff
@@ -121,7 +121,6 @@ const OptionalText = styled.span`
 `;
 
 const LabelContainer = styled.div`
-  display: flex;
```

Comment:
Removing this line will cause `align-items` to have no effect, which is a problem if there's an icon. Try to replace 

```
{iconName && (
  <StyledIcon
    name={iconName}
    ariaHidden={iconAriaHidden}
    title={!iconAriaHidden && (props as any)?.iconTitle}
  />
)}
```
with e.g. 
```
{true && (
  <StyledIcon
    name={'eye'}
    ariaHidden={iconAriaHidden}
    title={!iconAriaHidden && (props as any)?.iconTitle}
  />
)}

```
to see what I mean.

What you need instead is to wrap 

```
propsHasValues(props) ? (
  props.labelValue
) : (
  <FormattedMessage
    {...props.labelMessage}
    values={props.labelMessageValues}
  />
)}
```
with e.g. a `span`, then add margin-right of 3px to that styled span.

I'm not sure how the display: flex is causing this, but the solution I suggest above should fix it. :-)

Reply:
Ah I see - thank you!